client/js/components/fragments/Loading.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+
+import Logo from '../../../img/xteam-vertical.png';
+import 'file?name=[name].[ext]!../../../img/xteam-vertical.png';
+
+class Loading extends Component {
+  render() {
+    return (
+      <div id="loading" className="panel panel-homescreen">
```

Comment:
Can you set a fixed height to that component? On smaller screen a vertical scroll-bar appears there